src/Database/Schema/SqliteSchemaDialect.php

```diff
@@ -110,6 +110,9 @@ protected function _convertColumn(string $column): array
             return ['type' => TableSchemaInterface::TYPE_BOOLEAN, 'length' => null];
         }
 
+        if (($col === 'binary' && $length === 16) || (str_contains(strtolower($column), 'blob') && $col === 'uuid')) {
```

Comment:
Would it make more sense to simplify this?
```suggestion
        if (($col === 'binary' && $length === 16) || strtolower($column) === 'uuid_blob') {
```